website/docs/reference/global-configs/resource-type.md

```diff
@@ -109,3 +119,27 @@ Instead of targeting specific resources, use the `--resource-flag` or `--exclude
     </File>
 
 </VersionBlock>
+
+<VersionBlock firstVersion="1.9">
+
+-  In this example, use the following command to exclude _all_ unit tests when running tests. Note that the `--exclude-resource-type` flag is only available in dbt version 1.8 and higher:
```

Comment:
1.8 or 1.9?

Reply:
Oops, missed that during copy-paste -- good catch!

Removing the "dbt version x.y and higher piece since this is within a `<Version>` tag and will only show when Versionless or v1.9+ is chosen. 

```suggestion
-  In this example, use the following command to exclude _all_ unit tests when running tests:
```